En starting page

JonasBley opened the pull request:
Removed the math symbol from the starting page, added participate button and changed vision text. closes #390

arekkas commented:
Awesome!

arekkas commented on src/module/Ui/templates/layout/en/serlo-home.phtml:
```diff
@@ -80,68 +80,45 @@ echo $this->doctype();
     <div id="page">
         <section class="clearfix">
             <div id="content-layout" class="clearfix">
-                <?php echo $this->partial('layout/partials/notifications'); ?>
```
Why did you remove this?

arekkas replied:
@JonasBley it would be great if you could answer this :)

JonasBley replied:
I think I added it again in the latest version

arekkas replied:
No, it's still missing :)

JonasBley replied:
K second

arekkas commented on src/module/Ui/templates/layout/en/serlo-home.phtml:
```diff
@@ -80,68 +80,45 @@ echo $this->doctype();
                 <div class="home-serlo-row home-row">
                     <div class="home-column">
                         <div class="logo-wrapper">
-                            <?php echo $this->brand()->getLogo(); ?>
```
Why did you remove this? Isn't this equivalent to `<span class="serlo-logo">V</span> <div class="serlo-brand">Serlo</div> </div>`?

arekkas commented on src/module/Ui/templates/layout/en/serlo-home.phtml:
```diff
@@ -80,68 +80,45 @@ echo $this->doctype();
+                        <span style="padding: 15px; font-size: 16pt;"><a href="/23727">Participate <i class="fa fa-arrow-circle-right"></i></a></span>
                     </div>
                 </div>
-
```
Blank lines help in keeping readability, please add them again :)

arekkas commented:
Comments inline

arekkas commented:
Could you additionally create a screenshot, so I get a better feeling of what this looks like?

JonasBley commented:
![screenshot 2016-09-07 17 15 04](https://cloud.githubusercontent.com/assets/14056541/18317514/ba37153c-751e-11e6-80e4-783784f06d68.png)

arekkas commented:
What does it look like when you're logged in?

arekkas commented:
The default login is `admin@localhost` with password `123456` iirc

JonasBley commented:
I have  changed things according to what you said. The screenshot is also not up to date, will create a new pull request tomorrow

JonasBley commented:
Changed a small thing in the styles and need to update them on my machine as well, I will see tomorrow how it looks

inyono commented:
> [...] will create a new pull request tomorrow

Not needed, just push additional commits to the same branch.

JonasBley commented:
![screenshot 2016-09-08 13 35 41](https://cloud.githubusercontent.com/assets/14056541/18348020/3dcf8956-75c9-11e6-9ad8-95530ef5b8c1.png)

arekkas commented on src/assets/source/styles/partials/_serlo_home.scss:
```diff
@@ -70,6 +56,20 @@ $homeBreakpoint-lg-max: ($homeBreakpoint-lg-min - 1);
 }
 
+.home-emph-link {
+      font-size: 23px;
```
please use the standard identation, you can autoformat in most IDEs :)

arekkas commented on src/module/Ui/templates/layout/en/serlo-home.phtml:
```diff
@@ -80,68 +80,45 @@ echo $this->doctype();
-                            <?php echo $this->brand()->getLogo(); ?>
-                            <?php echo $this->brand()->getBrand(); ?>
-                        </div>
+                            <span class="serlo-logo">V</span> <div class="serlo-brand">Serlo</div> </div>
```
Could you put the trailing div into a new line?

```
                            <span class="serlo-logo">V</span> <div class="serlo-brand">Serlo</div>
                        </div>
```

JonasBley replied:
Oh that part was a bit messed up, sorry. Should be better now though I cant see how stuff looks right now, it says it rsyncs the changes, but they don't show up on my machine...

arekkas replied:
If that happens, kill the rsync process and restart it with `vagrant rsync-auto`

JonasBley replied:
I tried that, I'll try again

JonasBley replied:
rsync-auto is integrated in the vagrant up command or not?

arekkas commented:
It would be great if you could make a screenshot of the german landing page as well :)

JonasBley commented:
I didnt change anything on the German landing page

arekkas commented:
> I didnt change anything on the German landing page

You did, this PR contains 3 changed files, on of them is the german landing page: https://github.com/JonasBley/athene2/blob/f8ffafc1175e084d10ff7c06531c79a135e873ef/src/module/Ui/templates/layout/de/serlo-home.phtml

arekkas commented:
You can view all changes of your PR on GitHub:

![sdfgsgdf](https://cloud.githubusercontent.com/assets/3372410/18356466/065433ae-75ed-11e6-89d4-ad4d869aaa69.JPG)

JonasBley commented:
I created the class "home-emph-link" and moved the link properties from the class "home-column" to the new class, so I added all the links on the German page to the home-emph-link class. 
I needed to do that, because I couldnt otherwise change the participate and the about button on en.serlo.org

JonasBley commented:
I didnt actually change anything on the German page, it looks just the same

arekkas commented:
> I didnt actually change anything on the German page, it looks just the same

Haha, my bad, got it :)

arekkas commented on src/module/Ui/templates/layout/de/serlo-home.phtml:
```diff
@@ -184,16 +184,16 @@ echo $this->doctype();
                     <div class="home-community-link-row home-row">
                         <div class="home-column home-col-md-3">
-                            <a href="/19869" class="home-show-md">Mitmachen <i class="fa fa-arrow-circle-right"></i></a>
+                            <a href="/19869" class="home-emph-link" class="home-show-md">Mitmachen <i class="fa fa-arrow-circle-right"></i></a>
```
This should be `class="home-emph-link home-show-md"`

arekkas commented on src/module/Ui/templates/layout/de/serlo-home.phtml:
```diff
@@ -184,16 +184,16 @@ echo $this->doctype();
                         </div>
                         <div class="home-column home-col-md-3">
-                            <a href="/21565" class="home-show-md">Spenden <i class="fa fa-arrow-circle-right"></i></a>
+                            <a href="/21565" class="home-emph-link" class="home-show-md">Spenden <i class="fa fa-arrow-circle-right"></i></a>
```
This should be `class="home-emph-link home-show-md"`

arekkas commented on src/module/Ui/templates/layout/en/serlo-home.phtml:
```diff
@@ -143,11 +151,13 @@ echo $this->doctype();
+
                         <div class="home-column home-col-md-3">
-                            <a href="/27469" class="home-show-md">Participate <i class="fa fa-arrow-circle-right"></i></a>
+                            <a href="/27469" class="home-emph-link" class="home-show-md">Participate <i class="fa fa-arrow-circle-right"></i></a>
```
This should be `class="home-emph-link home-show-md"`

arekkas commented on src/module/Ui/templates/layout/en/serlo-home.phtml:
```diff
@@ -143,11 +151,13 @@ echo $this->doctype();
+
                         <div class="home-column home-col-md-3">
-                            <a href="/39837" class="home-show-md">Donate <i class="fa fa-arrow-circle-right"></i></a>
+                            <a href="/39837" class="home-emph-link" class="home-show-md">Donate <i class="fa fa-arrow-circle-right"></i></a>
```
This should be `class="home-emph-link home-show-md"`

JonasBley commented:
![screenshot 2016-09-08 19 22 04](https://cloud.githubusercontent.com/assets/14056541/18359294/95487370-75f9-11e6-9424-a429defc1f43.png)

arekkas commented:
Awesome! Thank you!
